probe/endpoint/reporter.go

```diff
@@ -71,6 +74,11 @@ func (r *Reporter) Stop() {
 	r.scanner.Stop()
 }
 
+type fourTuple struct {
+	fromAddr, toAddr string
+	fromPort, toPort uint16
+}
```

Comment:
Now we have this type, we could replace the arguments to addConnection with an instance of this?